code/__DEFINES/mobs.dm

```diff
@@ -311,6 +311,8 @@ GLOBAL_LIST_INIT(xenoupgradetiers, list(XENO_UPGRADE_BASETYPE, XENO_UPGRADE_INVA
 #define IS_SYNTHETIC 			(1<<14)
 #define NO_STAMINA 				(1<<15)
 #define DETACHABLE_HEAD			(1<<16)
+#define IS_PSYCHIC				(1<<17)
```

Comment:
Should rename this to IS_SECTOID_ALIEN since it has nothing to do with psychic abilities and is just used for the gun check.
Alternatively just give them psychic abilities stolen from the shrike